app/components/Preview/Types/PreviewUri.tsx

```diff
@@ -12,11 +13,14 @@ export type PreviewUriProps = {
 };
 
 export function PreviewUri(props: PreviewUriProps) {
-  const previewFetcher = useFetcher<PreviewResult>();
+  const { isOnline, onceOnline } = useNetworkState();
+  const previewFetcher = useFetcher < PreviewResult > ();
```

Comment:
Can you rewrite this to be `useFetcher<PreviewResult>();`?

Reply:
Sorry for the formatting issue, just remarked that's the result of my code editor formatting.